src/Platform/Microsoft.Testing.Platform/Messages/TestNodeUpdateMessage.cs

```diff
@@ -1,14 +1,35 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Text;
+
 using Microsoft.Testing.Platform.TestHost;
 
 namespace Microsoft.Testing.Platform.Extensions.Messages;
 
 public sealed class TestNodeUpdateMessage(SessionUid sessionUid, TestNode testNode, TestNodeUid? parentTestNodeUid = null)
-    : DataWithSessionUid(nameof(TestNodeUpdateMessage), "This data is used to report a TestNode state change.", sessionUid)
+    : DataWithSessionUid("TestNode update", "This data is used to report a TestNode state change.", sessionUid)
```

Comment:
this is an "id" should we have something more "technical" in case it's saved somewhere?

Reply:
oh so ok I thought it was the id, never mind.